run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
I would suggest to turn this into Makefile. And default command for make could be a build.

So we could have: make build and make run.

Reply:
You're right, I will look into Makefile rather than a shell script.